src/app/components/LiveHeaderMedia/index.test.tsx

```diff
@@ -263,4 +196,93 @@ describe('liveMediaStream', () => {
       });
     });
   });
+
+  describe('Text content states', () => {
+    it.each([
+      {
+        title:
+          'Displays correct text content in open state with no punctuation',
+        inputTitle: 'Title with no punctuation',
+        expectedResult:
+          'Title with no punctuation, Contains some upsetting scenes.Watch',
+      },
+      {
+        title: 'Displays correct text content in open state with punctuation',
+        inputTitle: 'Title with punctuation!',
+        expectedResult:
+          'Title with punctuation!, Contains some upsetting scenes.Watch',
```

Comment:
I am now wondering if we expect this or not - I am not sure if `!,` together is desired. @greenc05 is this something you could advise us on?

```suggestion
          'Title with punctuation!, Contains some upsetting scenes.Watch',
```

Reply:
Just to clarify, `,` would be visually hidden, but create a "pause" effect, correct?

Reply:
Yes that's correct - the entire phrase will be visually hidden, but I did wonder punctuation such as ! already creates a pause effect in screenreaders?

Reply:
! followed by comma is slightly odd, as is no space before the word 'Watch'. Correct english / grammar is all that's required.

'Title with punctuation! Contains some upsetting scenes. Watch',